ui/desktop/app/styles/app.scss

```diff
@@ -662,3 +662,31 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
   gap: 0.5rem;
   width: inherit;
 }
+
+// Settings
+.settings-card {
+  .hds-card__container {
+    padding: 1rem;
+  }
+
+  .header-and-icon {
+    display: flex;
+    align-items: center;
+    gap: 0.5rem;
+  }
+
+  .content {
+    padding-left: 2.2rem;
+
+    .hds-form-field--layout-vertical {
```

Comment:
Is this the right selector? It seems like we'll need margin for multiple things in different cards and this is very specific to the theme selector